docs/source/development/creating-views.md

```diff
@@ -135,20 +136,21 @@ export { FullView };
 
 ## Registering The View
 
-To register the view we will edit the `config.js` file. The `views`
-configuration options contains all the views. This object contains an object
-called `layoutViews` which registers all the layout views. We will add the `full_view` to this object.
+To register the view we will edit the `config.js` file.
```

Comment:
```suggestion
To register the view we will edit the `index.js` file.
```

Reply:
@plone/volto-team please review for accuracy.